### PR TITLE
Feature/django32python310

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: resolwe/flow/executors/startup_processing_container.py [hunk 1]
```diff
@@ -344,8 +344,13 @@ def __init__(self, loop):
         self.exported_files_mapper = dict()  # type: Dict[str, str] # noqa: F821
         self.upload_socket = None
         self.update_logs_future = None
-        self._uploading_log_files_lock = asyncio.Lock(loop=loop)
-        self._send_file_descriptors_lock = asyncio.Lock(loop=loop)
+        # Drop this when we stop supporting Python 3.7.
+        # This class has been implicitly getting the current running loop since 3.7.
+        lock_arguments = {}
+        if sys.version_info < (3, 7):
```
Comment (dblenkus) [inline resolwe/flow/executors/startup_processing_container.py:350]: The comment says 3.7, but here it is 3.6.

File: tests/settings.py [hunk 2]
```diff
@@ -59,6 +59,8 @@
     "resolwe.permissions.permissions.ResolwePermissionBackend",
 )
 
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```
Comment (dblenkus) [inline tests/settings.py:61]: This is already the default value: https://docs.djangoproject.com/en/4.0/releases/3.2/#customizing-type-of-auto-created-primary-keys

File: setup.py [hunk 3]
```diff
@@ -38,46 +38,43 @@
...
         # Storage requirement for computing hashes.
         "crcmod",
-        "kubernetes~=12.0.1",
-        "docker~=4.4.4",
+        "kubernetes~=21.7.0",
+        "docker~=5.0.3",
         "Django~=3.2.12",
-        "djangorestframework~=3.12.2",
-        "django-filter~=2.4.0",
-        "django-guardian~=2.3.0",
```
Comment (dblenkus) [inline setup.py:56]: It would be nicer to move this line into the last commit.

Comment (gregorjerse) [reply setup.py:56]: Ah, sure.

File: resolwe/flow/executors/startup_processing_container.py (hunk 1 above)
Comment (gregorjerse) [reply resolwe/flow/executors/startup_processing_container.py:350]: The comment is wrong, fixed.

File: tests/settings.py (hunk 2 above)
Comment (dblenkus) [reply tests/settings.py:61]: Either of them is fine, but I would go with the second one to follow Django.

**Review by dblenkus — APPROVED** [review]

File: tests/settings.py (hunk 2 above)
Comment (gregorjerse) [reply tests/settings.py:61]: Sure, that is better in the long run.

File: resolwe/process/models.py [hunk 4]
```diff
@@ -394,6 +395,7 @@ def _get_field_data(self, field: Field) -> Any:
 
         :raises RuntimeError: when no data is received.
         """
+        print("Getting field data for field", field)
```
Comment (dblenkus) [inline resolwe/process/models.py:398]: Remove the print.

**Pull request merged**